app/views/foreman_tasks/tasks/dashboard/_latest_tasks_in_error_warning.html.erb

```diff
@@ -11,7 +11,7 @@
       <td class="ellipsis"><%= link_to task.humanized[:action], defined?(main_app) ? main_app.foreman_tasks_task_path(task.id) : foreman_tasks_task_path(task.id) %></td>
       <td><%= task.state %></td>
       <td><%= task.result %></td>
-      <td><%= _('%s ago') % time_ago_in_words(task.ended_at) %></td>
+      <td><%= task.ended_at ? _('%s ago') % time_ago_in_words(task.ended_at) : _('Paused') %></td>
```

Comment:
I prefer not having an empty column, would "N/A" make sense? btw, is there any way to tell when the task was paused? that would be better then not showing anything.